oshi-core/src/main/java/oshi/software/os/windows/WindowsOperatingSystem.java

```diff
@@ -56,11 +56,11 @@
     // For WMI Process queries
     private static String processProperties = "Name,ExecutablePath,CommandLine,ExecutionState,ProcessID,ParentProcessId"
             + ",ThreadCount,Priority,VirtualSize,WorkingSetSize,KernelModeTime,UserModeTime,CreationDate"
-            + ",ReadTransferCount,WriteTransferCount,__PATH,__PATH";
+            + ",ReadTransferCount,WriteTransferCount,__PATH,__PATH,HandleCount";

```

Comment:
purely preference, I'd prefer to move the HandleCount before the two process queries, to leave them at the end...

Reply:
so after `WriteTransferCount`?

Reply:
Yep... again, purely preference to make it easier to find those processes at the end.